src/css/tabs/pid_tuning.css

```diff
@@ -488,7 +488,7 @@
 }
 
 .tab-pid_tuning .resetbt {
-    width: 140px;
+    width: 200px;
```

Comment:
Maybe `fit-content` would work also (if we have different messages, otherwise ignore)